t/live.t

```diff
@@ -201,7 +201,7 @@ Charges: {
             );
         } 'Created a charge object with metadata';
         isa_ok $charge, 'Net::Stripe::Charge';
-        ok defined($charge->metadata), "charge has metadata";        
+        ok defined($charge->metadata), "charge has metadata";
```

Comment:
Thanks for white space clean-up, but in the future please do that as a separate commit so that reverting this change won't collide with other code making logical changes in this area.